src/app/login/login.service.ts

```diff
@@ -32,8 +32,12 @@ export class LoginService {
           observable.next(user);
           observable.complete();
         },
-        (error: HttpErrorResponse) => {
-          this.alertService.detailedError(error);
+        (error: HttpErrorResponse | {}) => {
+          if ('status' in error) {
```

Comment:
Suggest a comment here and in register.service.ts as to why you're doing this. We all know now why from the PR description, but it might not be obvious a year from now to somebody new looking at the code.

Reply:
Added some comments.